src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -187,29 +180,41 @@ public WindowImpl()
             s_instances.Add(this);
         }
 
-        public Action Activated { get; set; }
+        /// <inheritdoc />
```

Comment:
WindowImpl shouldn't be a public API, so these inheritdocs are not needed here.

We need to make WindowImpl internal soon.

Reply:
Removed all!

Reply:
We probably should reach agreement on the inheritdocs tag sometime. I think it's useful for consistency, readability and also some IDE functionality. I know @maxkatz6 likes to remove them.

Reply:
According to this review and https://github.com/AvaloniaUI/Avalonia/pull/10256#discussion_r1099871290, I think the convention is:

- No docs on non-public APIs
- On public APIs, `<inheritdoc />` should be used in all places where it makes sense

Reply:
... and we should standardize on the space or no space. Space was required for really old parsers.
Visual Studio won't add it by default I think.

But yea, that seems to be what is done here. I normally would expect XML docs to be the same in cases like this though. Or at least anything that isn't private.

Reply:
@robloo previously I thought that SDK can inherit documentation properly. In which case I would prefer removing inheritdoc from everywhere, except cases where you need to specify another member to inherit (using "cref" attr).

But apparently, I was wrong, and documentation inheriting is only an IDE feature and not properly integrated into the SDK. So... yes, we need to keep it then.